src/data_provider/src/packages/packagesWindows.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Wazuh SYSINFO
+ * Copyright (C) 2015-2021, Wazuh Inc.
```

Comment:
According to this PR https://github.com/wazuh/wazuh/pull/11780/, current year is not necessary anymore
```suggestion
 * Copyright (C) 2015, Wazuh Inc.
```